app/views/pregnancies/_new_pregnancy.html.erb

```diff
@@ -27,8 +27,16 @@
 
   <div class="col-sm-2">
     <%= f.form_group :voicemail_ok, label: { text: 'Voicemail OK?' } do %>
-    <%= f.check_box :voicemail_ok, label: 'Yes' %> <!-- TODO add bootstrap toggle -->
+      <%= f.check_box :voicemail_ok, label: 'Yes' %> 
     <% end %>
+    <!-- This is one attempt at the bootstrap toggle -->
+    <%# f.form_group :voicemail_ok, label: { text: 'Voicemail OK?' }, data: {toggle: 'toggle', on: 'Yes', off: 'No' } do %>
```

Comment:
hey! So I think what's going on here can be best summarized as: javascript is awful. 

When you pop in new HTML via ajax, javascript doesn't necessarily execute on it unless you specifically tell it to. So the page loads, the bootstrap toggle script fires, and then it just sits there. It continues sitting there even as new data-toggle elements get added. 

So basically, the fix here is to add `<script>$('[data-toggle="toggle"]').bootstrapToggle();</script>` to the bottom of the new pregnancy partial, which will tell the check boxes with data-toggle to be toggle switches. 

I have lost what feels like years of my life to this very problem! Sorry you ran into it.

In conclusion, javascript :(